Develop/scripts/script.js

```diff
@@ -0,0 +1,198 @@
+$(document).ready(function () {
+  var cityArray = [];
```

Comment:
Probably worth looking at let and const. Since 2015 people have moved to these generally, modern JS you wouldn't use var much.